templates/UmbracoProject/UmbracoProject.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Umbraco.Cms.Targets" Version="UMBRACO_VERSION_FROM_TEMPLATE" />
```

Comment:
This should not be nessasary as it is referenced by Umbraco.Cms already

Reply:
This reference ensures that the `appsettings-schema.json` and `umbraco-package-schema.json` files are placed in the root of the project. Without this reference, the files simply do not show up.

If that is the case, that Umbraco.Cms already has this reference, then something is missing somewhere, because the files are not present if you install with `dotnet new umbraco`.

Reply:
Oh I see, will just test again.